components/ai_chat/core/common/BUILD.gn

```diff
@@ -19,6 +19,7 @@ static_library("common") {
     "//base",
     "//brave/components/ai_chat/core/common/buildflags:buildflags",
     "//components/prefs",
+    "//components/user_prefs",
```

Comment:
this is wrong. please just move `pref_names.h/cc` into `browser` target.

Reply:
thanks, it's removed.